docs/about/compatibility/windows-support.md

```diff
@@ -68,11 +68,13 @@ on Windows is provided with two levels on enablement.
 
 ### Support status
 
-✅: **Supported** - AMD enables these GPUs in our software distributions for
-  the corresponding ROCm product.
+✅: **Supported** - AMD enables these GPUs in our software
+distributions for the corresponding ROCm product.
+
 ⚠️: **Deprecated** - Support will be removed in a future release.
```

Comment:
Deprecated is sometimes a future warning. We have had instances where APIs are deprecated in a current release. 

@saadrahim